exercise-book/src/nrf52-radio-using-hal.md

```diff
@@ -10,25 +10,25 @@ let board = dk::init().unwrap();
 
 This grants you access to the board's peripherals, like its LEDs.
 
-The `dk` crate / library is a Board Support Crate tailored to this workshop to make accessing the peripherals used in this workshop extra seamless. You can find its source code at `nrf52-code/boards/dk/src/`.
+The `dk` crate / library is a Board Support Package tailored to this workshop to make accessing the peripherals used in this workshop extra seamless. You can find its source code at `nrf52-code/boards/dk/src/`.
 
-`dk` is based on the [`nrf52840-hal`] crate, which is a Hardware Abstraction Layer (HAL) over the nRF52840 Development Kit. The purpose of a HAL is to abstract away the device-specific details of the hardware, for example registers, and instead expose a higher level API more suitable for application development.
+`dk` is based on the [`nrf52840-hal`] crate, which is a Hardware Abstraction Layer (HAL) over the nRF52840 System on Chip. The purpose of a HAL is to abstract away the device-specific details of the hardware, for example registers, and instead expose a higher level API more suitable for application development.
 
 The `dk::init` function we have been calling in all programs initializes a few of the nRF52840 peripherals and returns a `Board` structure that provides access to those peripherals. We'll first look at the `Leds` API.
 
 ✅ Run the `led` program. Two of the green LEDs on the board should turn on; the other two should stay off.
 
 > NOTE this program will not terminate itself. Within VS code you need to click "Kill terminal" (garbage bin icon) in the bottom panel to terminate it.
 
-✅ Open the documentation for the `dk` crate by running the following command from the `nrf52-code/radio` folder:
+✅ Open the documentation for the `dk` crate by running the following command from the `nrf52-code/radio-app` folder:
 
 ```console
 cargo doc -p dk --open
 ```
 
 ✅ Check the API docs of the `Led` abstraction. Change the `led` program, so that the bottom two LEDs are turned on, and the top two are turned off.
 
-🔎 If you want to see logs from Led API of the `dk` Hardware Abstraction Layer, flash the dk with the following environment variable:
+🔎 If you want to see logs from Led API of the `dk` Board Support Package, flash the dk with the following environment variable:
```

Comment:
Could you please add the place where these logs are dumped to? I  don't think I'm able to find them in the `target/` directory, but it would be neat to save trainees of that hassle.

Reply:
They come from the output of probe-run / probe-rs.

Reply:
Added a note